src/data/finset/pointwise.lean

```diff
@@ -760,6 +762,9 @@ lemma smul_finset_singleton (b : β) : a • ({b} : finset β) = {a • b} := im
 @[to_additive] lemma smul_finset_inter_subset : a • (s₁ ∩ s₂) ⊆ a • s₁ ∩ (a • s₂) :=
 image_inter_subset _ _ _
 
+@[simp] lemma bUnion_smul_finset (s : finset α) (t : finset β) : s.bUnion (• t) = s • t :=
+bUnion_image_left
```

Comment:
What about the right variant, `t.bUnion ((•) s) = s • t`? Is that also true?

Reply:
It does not make sense (and your statement does not typecheck) because we can multiply `a : α` by `s : set α` only on the left.